tests/Unit/Domain/CoordinateMaps/TimeDependent/ShapeMapTransitionFunctions/Test_Wedge.cpp

```diff
@@ -0,0 +1,350 @@
+        outer_sphericity, this_wedge_inner_radius, this_wedge_outer_radius,
+        middle_sphericity, outer_sphericity, orientation);
+
+    TimeDependent::Shape shape{center, l_max, l_max, std::move(wedge),
+                               fot_name};
+
+    // Test 10 points for each sphericity and orientation
+    for (size_t i = 0; i < 10; i++) {
+      const double radius = radial_dist(generator);
+      const double theta = theta_dist(generator);
+      const double phi = phi_dist(generator);
+
+      CAPTURE(radius);
+      CAPTURE(theta);
+      CAPTURE(phi);
+
+      const std::array<double, 3> centered_z_aligned_point =
+          sph_to_cart(radius, theta, phi);
+      const std::array<double, 3> centered_point =
+          discrete_rotation(orientation, centered_z_aligned_point);
+      const std::array<double, 3> point = centered_point + center;
+      CAPTURE(centered_z_aligned_point);
+      CAPTURE(centered_point);
+      CAPTURE(point);
+
+      const std::array<double, 3> mapped_point =
+          shape(point, initial_time, functions_of_time);
+      const std::optional<std::array<double, 3>> inverse_mapped_point =
+          shape.inverse(mapped_point, initial_time, functions_of_time);
+
+      CAPTURE(mapped_point);
+      CAPTURE(inverse_mapped_point);
+
+      CHECK(inverse_mapped_point.has_value());
+      CHECK_ITERABLE_APPROX(point, inverse_mapped_point.value());
+
+      const auto check_bad_point = [&](const std::string& info,
+                                       const double inner_bound,
+                                       const double outer_bound) {
+        INFO(info);
+        std::uniform_real_distribution<double> bad_radial_dist{inner_bound,
+                                                               outer_bound};
+        // Want theta to be from 0 to pi, so just use phi and divide by 2
+        const double random_theta = phi_dist(generator) / 2.0;
+        const double random_phi = phi_dist(generator);
+        const double bad_radius = bad_radial_dist(generator);
+
+        const std::array<double, 3> bad_point =
+            sph_to_cart(bad_radius, random_theta, random_phi) + center;
+
+        CAPTURE(bad_point);
+        CHECK_FALSE(shape.inverse(bad_point, initial_time, functions_of_time)
+                        .has_value());
+      };
+
+      // The 0.9 factor guarantees that we are inside the mapped outer radius of
+      // the inner wedge.
+      check_bad_point("Point in inner wedge", overall_inner_radius,
+                      0.9 * this_wedge_inner_radius / sqrt(3.0));
+      check_bad_point("Too small radius", 0.01,
+                      overall_inner_radius / sqrt(3.0));
+      check_bad_point("Too big radius", this_wedge_outer_radius * 10.0,
+                      this_wedge_outer_radius * 50.0);
+    }
+  }
+}
+
+SPECTRE_TEST_CASE("Unit.Domain.CoordinateMaps.Shape.Wedge", "[Domain][Unit]") {
+  // test_only_transition();
+  (void)test_only_transition;
```

Comment:
Looks like you disabled this for debugging? Also the other test function should be called.